hooks/field/useDatepickerField.ts

```diff
@@ -0,0 +1,171 @@
+// Copyright (c) 2022. Heusala Group Oy <info@heusalagroup.fi>. All rights reserved.
+
+import { useCallback, useEffect, useState } from "react";
+import { FormFieldState } from "../../types/FormFieldState";
+import { useFieldChangeState } from "./useFieldChangeState";
+import { LogService } from "../../../core/LogService";
+import { useFieldDateChangeEventCallback } from "./string/useFieldStringChangeEventCallback";
+import { useFieldIdentifier } from "./useFieldIdentifier";
+import { useFieldMountEffectWithInternalState } from "./useFieldMountEffectWithInternalState";
+import { FieldChangeCallback } from "./useFieldChangeCallback";
+import moment from "moment-timezone";
```

Comment:
Same note about `TimeService`

Reply:
Added new method to TimeService so imports can be removed